x-pack/test/functional/apps/painless_lab/painless_lab_flyout.ts

```diff
@@ -21,9 +38,24 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('click show API request button and flyout should appear in page', async () => {
+      // replace the default script with a simpler one
+      await monacoEditor.setCodeEditorValue(TEST_SCRIPT);
```

Comment:
Have you thought about moving this bit into the `before`  block? From my point of view, it's more of a test preparation than actually part of the `click show API request button` test.

Reply:
Good point, moved into _before_ method [7874ad1](https://github.com/elastic/kibana/pull/193851/commits/7874ad1d0396c356aca17e913e424b6d8fca90d6)